packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -804,7 +805,14 @@ export class FluidDataStoreRuntime
 						trackState,
 						telemetryContext,
 					);
-					summaryBuilder.addWithStats(contextId, contextSummary);
+					// Add the context's summary to the summary tree. Also, prefix all handles in its summary tree with
+					// "channelsTreeName" name and its id since that is where its previous summary would be. The context
+					// only adds the handle id relative to its path.
+					summaryBuilder.addWithStatsAndPrefixHandles(
+						`${channelsTreeName}/${contextId}`,
```

Comment:
Even though the value is the same, should be use different variable names for "channels" name used at container runtime and data store levels?

Reply:
We use the same name when we put the summaries into the "channels" sub-tree [here](https://github.com/microsoft/FluidFramework/blob/2adb773556897202d4e68f80083fc4b770bd7ec7/packages/runtime/container-runtime/src/containerRuntime.ts#L2712) and [here](https://github.com/microsoft/FluidFramework/blob/2adb773556897202d4e68f80083fc4b770bd7ec7/packages/runtime/container-runtime/src/dataStoreContext.ts#L567). So, I just used the same here.